doc/overview/project_capabilities.md

```diff
@@ -12,6 +12,15 @@ known to declare the capability.
 | VisualC | Project may contain or compile C++ source files.
 | CSharp | Project may contain or compile C# source files.
 | VB | Project may contain or compile VB source files.
+| JavaScript | Project may contain JavaScript source files.
```

Comment:
Any project may contain Javascript or TypeScript files. That's not really a capability. What does this project capability really mean? Does it mean the project will execute these javascript files as its primary function?